### PR TITLE
MM_34997: disabled user consent option in app install dialog

### PR DESCRIPTION
**Pull request by levb**
https://mattermost.atlassian.net/browse/MM-34997

The setting is not relevant yet (well, neither is creating the OAuth2 app yet, but...)

### CODE REVIEW

File: server/http/dialog/install.go [hunk 1]
```diff
@@ -47,25 +47,25 @@ func NewInstallAppDialog(m *apps.Manifest, secret, pluginURL string, commandArgs
...
-					Value: "require",
-				},
-				{
-					Text:  "Do not require user consent",
-					Value: "notrequire",
-				},
-			},
-		})
-	}
+	// if m.RequestedPermissions.Contains(apps.PermissionActAsUser) {
```
Comment (catalintomai) [inline server/http/dialog/install.go:50]: maybe add a comment/give context wrt to the future plans for the commented-out code.

**Review by catalintomai — APPROVED** [review]

File: server/http/dialog/install.go (hunk 1 above)
Comment (hanzei) [reply server/http/dialog/install.go:50]: Is there a ticket we can link?

**Review by hanzei — APPROVED** [review]
LGTM

**Review by DHaussermann — APPROVED** [review]
Tested and passed 
- Tested various apps (http-hello and Service Now)
- The option was removed as expected
- App function normally after instal no issue found

LGTM!

**Pull request merged**